easybuild/easyconfigs/g/GATK/GATK-4.0.1.2-Java-1.8.0_92.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'Tarball'
+
+name = 'GATK'
+version = '4.0.1.2'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://www.broadinstitute.org/gatk/'
+description = """The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
+ to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
+ with a primary focus on variant discovery and genotyping as well as strong emphasis on
+ data quality assurance. Its robust architecture, powerful processing engine and
+ high-performance computing features make it capable of taking on projects of any size."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/broadinstitute/gatk/releases/download/%(version)s/']
+sources = ['%(namelower)s-%(version)s.zip']
+checksums = ['d667c08ec44f4dc2029d00ca16cfcfe7850ae9bfdcdd6e35f3048b8e7e83647b']
+
+dependencies = [('Java', '1.8.0_92')]
```

Comment:
Use the `Java/1.8` wrapper module here rather than fixing to a specific `Java` version?

```python
dependencies = [('Java', '1.8')]
```

(see also https://easybuild.readthedocs.io/en/latest/Wrapping_dependencies.html)